tests/raster/test_workingcopy.py

```diff
@@ -527,3 +528,110 @@ def _set_category_labels(pam_path, category_labels):
                     category.firstChild.nodeValue = category_labels[row_id]
 
         pam_path.write_text(parsed.toxml())
+
+
+def test_working_copy_edit__convert_to_cog(
+    cli_runner, data_archive, requires_gdal_info, requires_git_lfs, check_lfs_hashes
+):
+    with data_archive("raster/tif-aerial.tgz") as tif_aerial:
+        with data_archive("raster/aerial.tgz") as repo_path:
+            r = cli_runner.invoke(["diff", "--exit-code"])
+            assert r.exit_code == 0
+
+            shutil.copy(tif_aerial / "aerial.tif", repo_path / "aerial/aerial2.tif")
+
+            assert get_hash_and_size_of_file(repo_path / "aerial/aerial2.tif") == (
+                "bdbb58a399b60231f7a017fd76659efb0f5c1d82ab892248123d14d9a1e838e1",
+                393860,
+            )
+
+            r = cli_runner.invoke(["status"])
+            assert r.exit_code == 0, r.stderr
+            assert r.stdout.splitlines() == [
+                "On branch main",
+                "",
+                "Changes in working copy:",
+                '  (use "kart commit" to commit)',
+                '  (use "kart restore" to discard changes)',
+                "",
+                "  aerial:",
+                "    meta:",
+                "      1 updates",
```

Comment:
why's there an update?

Reply:
Hmm. Basically because we are going from
```
- {
-   "fileType": "geotiff",
-   "profile": "cloud-optimized"
- }
```
to... something else - more on this below in next comment. This is not special-cased in any way, so it's an update.
Maybe we should call it something else... `kart status` now has `inserts` `updates` `deletes` and `spatialFilterConflicts` - this could be a `formatConflict` or `formatViolation` or something